### PR TITLE
update for silver JSON prod

### PR DESCRIPTION
**Pull request by jshlee**
updated JEC
updated PU to silver json

### CODE REVIEW

**Comment by jhgoh** [comment]
Shouldn't we close the 744 prod campaign and open new one? There are lots of changes from the last weeks.

File: CatProducer/prod/LumiMask/getPileUpData.py [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
+    tt = ROOT.TFile(PileUpData)
     histo = tt.Get("pileup")
-    print f
-    for b in range(histo.GetNbinsX()):
+    print '#%s'%command
+    print '"%s%s":cms.vdouble('%(puname,f)
+    for b in range(histo.GetNbinsX()+1):
         if b == 0:
-            continue        
+            continue
```
Comment (jhgoh) [inline CatProducer/prod/LumiMask/getPileUpData.py]:
``` python
for b in range(1, histo.GetNbinsX()+1):
```

**Comment by jshlee** [comment]
Hi @jhgoh 
is 69900 for minBiasXsec correct? and the 5%error?
also is there anything else to add for 745 prod?

**Comment by jshlee** [comment]
@jhgoh, can u comment on the minBiasXsec?

**Comment by jhgoh** [comment]
@jshlee I took 69900 from the slides, +-5% from the run I recipe.

**Comment by jhgoh** [comment]
https://indico.cern.ch/event/456951/contribution/17/attachments/1180309/1709353/tt_uev_preapp_talk.pdf
 page 9.

**Comment by jhgoh** [comment]
Hmm.. but not clear where is the source of this number.
This HN just says 69mb as the best input value.
https://hypernews.cern.ch/HyperNews/CMS/get/physics-validation/2451/1/3/2/1/1/1/1/1/1/1/1/1/1/1/1/1/1/1/2.html

**Comment by jshlee** [comment]
i've updated to 69mb

**Pull request merged**